receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -51,11 +58,46 @@ func (s *brokerScraper) scrape(context.Context) (pmetric.Metrics, error) {
 		s.client = client
 	}
 
+	now := pcommon.NewTimestampFromTime(time.Now())
+	rb := s.mb.NewResourceBuilder()
+	rb.SetClusterAlias(s.config.ClusterAlias)
+
 	brokers := s.client.Brokers()
+	s.mb.RecordKafkaBrokersDataPoint((now), int64(len(brokers)))
 
-	s.mb.RecordKafkaBrokersDataPoint(pcommon.NewTimestampFromTime(time.Now()), int64(len(brokers)))
+	if s.clusterAdmin == nil {
+		admin, err := newClusterAdmin(s.config.Brokers, s.saramaConfig)
+		if err != nil {
+			s.settings.Logger.Error("Error creating kafka client with admin priviledges", zap.Error(err))
+			return s.mb.Emit(metadata.WithResource(rb.Emit())), scrapeErrors.Combine()
+		}
+		s.clusterAdmin = admin
+	}
+
+	for _, broker := range brokers {
+		ID := strconv.Itoa(int(broker.ID()))
+		configEntries, err := s.clusterAdmin.DescribeConfig(sarama.ConfigResource{
+			Type:        sarama.BrokerResource,
+			Name:        ID,
+			ConfigNames: []string{"log.retention.hours"},
```

Comment:
I'm a bit confused, is this string a kafka-specific value? Unrelated to the metrics in this receiver? The value is being used to create a metric with a different name, so I just want to make sure I'm following this properly.

Reply:
Yes this string (and similar ones for topic) is a inbuilt config of a Kafka broker, We are just fetching this configs from kafka. I have avoided adding "hours" as metric name, Just to make it compliant with otel simentics. The unit for all time metrics is "s".